encoding.bs

```diff
@@ -845,7 +845,7 @@ specification, excluding <a>index single-byte</a>, which have their own table:
   <td colspan=3><a href=index-gb18030-ranges.txt>index-gb18030-ranges.txt</a>
   <td>This <a>index</a> works different from all others. Listing all code points would result
   in over a million items whereas they can be represented neatly in 207 ranges combined with trivial
-  limit checks. It therefore only superficially matches the GB18030-2005 standard for code points
+  limit checks. It therefore only superficially matches the GB18030-2022 standard for code points
```

Comment:
Should there be a remark about retaining 18 GB18030-2005 mappings in the index?

Reply:
Hmm. Since index-gb18030-ranges.txt does not get entry changes, perhaps the remark here should more clearly explain why -ranges.txt matches GB18030-2005 and not -2022.

Reply:
Oh right. It does match 2005 because for decoding we want the 2005 non-PUA mappings as they are more compatible and for encoding we instead prefer to map to the two-byte mappings using the encoder side table.

Reply:
So instead of

> It therefore only superficially matches the GB18030-2022 standard for code points encoded as four bytes.

maybe something like

> It therefore only superficially matches the GB18030-2005 standard for code points encoded as four bytes. It does not match the GB18030-2022 standard as that would increase the number of four-byte sequences mapping to Unicode Private Use Areas. The other direction is handled by a side table in the gb18030 encoder.

Reply:
LGTM.